.github/workflows/manifold.yml

```diff
@@ -107,6 +107,9 @@ jobs:
     timeout-minutes: 30
     runs-on: ubuntu-22.04
     if: github.event.pull_request.draft == false
+    strategy:
+      matrix:
+        exception: [ON, OFF]
```

Comment:
Is there any reason to build WASM with exceptions off? Maybe better to test one of the nix builds since they're fast?

Reply:
I have a use for wasm with exceptions off.

Reply:
@fire I think your earlier comment mentioned you do build WASM with exceptions off - that requires special CMake instructions which I was considering removing since JS is generally exceptions-based. But if WASM sans exceptions is actually useful to someone, then @pca006132 has it set up properly now. Do you build for WASM without exceptions?

Reply:
I would expect to build Godot Engine WASM without exceptions since the Godot Engine editor and Godot Engine game templates build without exceptions in the default configuration on all platforms.

Reply:
Thanks! Okay, let's leave it as is in this PR then.